File: thain-core/src/main/java/com/xiaomi/thain/core/mapper/JobMapper.java
/*
 * Copyright (c) 2019, Xiaomi, Inc.  All rights reserved.
 * This source code is licensed under the Apache License Version 2.0, which
 * can be found in the LICENSE file in the root directory of this source tree.
 */
package com.xiaomi.thain.core.mapper;

import com.xiaomi.thain.common.model.dr.JobDr;
import org.apache.ibatis.annotations.Param;

import java.util.List;

/**
 * Date 19-5-17 下午5:22
 *
 * @author liangyongrui@xiaomi.com
 */
public interface JobMapper {

    List<JobDr> getJobs(@Param("flowId") long flowId);
}

File: thain-core/src/main/kotlin/com/xiaomi/thain/core/mapper/JobMapper.kt
package com.xiaomi.thain.core.mapper

import com.xiaomi.thain.core.model.dr.JobDr
import org.apache.ibatis.annotations.Param

/**
 * Date 19-5-17 下午5:22
 *
 * @author liangyongrui@xiaomi.com
 */
interface JobMapper {

    fun getJobs(@Param("flowId") flowId: Long): List<JobDr>

    fun cleanUpExpiredAndDeletedJob(dataReserveDays: Int): Int
}
